skrifa/src/metrics.rs

```diff
@@ -116,29 +116,29 @@ impl Metrics {
             ..Default::default()
         };
         let coords = location.into().coords();
-        let scale = size.linear_scale(metrics.units_per_em);
+        let scale = ScaleFactor(size.fixed_linear_scale(metrics.units_per_em));
```

Comment:
Nit: Could the name `scale` and `ScaleFactor` include a hint that we're talking about `Fixed` type computations?

Reply:
Indeed. I’ll update the names